src/HumiditySensorDevice.ts

```diff
@@ -0,0 +1,213 @@
+
+// From DeviceAttestationCredsExample.cpp
+const CertificateDeclaration = ByteArray.fromHex("3082021906092a864886f70d010702a082020a30820206020103310d300b06096086480165030402013082017106092a864886f70d010701a08201620482015e152400012501f1ff3602050080050180050280050380050480050580050680050780050880050980050a80050b80050c80050d80050e80050f80051080051180051280051380051480051580051680051780051880051980051a80051b80051c80051d80051e80051f80052080052180052280052380052480052580052680052780052880052980052a80052b80052c80052d80052e80052f80053080053180053280053380053480053580053680053780053880053980053a80053b80053c80053d80053e80053f80054080054180054280054380054480054580054680054780054880054980054a80054b80054c80054d80054e80054f80055080055180055280055380055480055580055680055780055880055980055a80055b80055c80055d80055e80055f80056080056180056280056380182403162c04135a494732303134325a423333303030332d32342405002406002507942624080018317d307b020103801462fa823359acfaa9963e1cfa140addf504f37160300b0609608648016503040201300a06082a8648ce3d04030204473045022024e5d1f47a7d7b0d206a26ef699b7c9757b72d469089de3192e678c745e7f60c022100f8aa2fa711fcb79b97e397ceda667bae464e2bd3ffdfc3cced7aa8ca5f4c1a7c");
+
+Network.get = singleton(() => new NetworkNode());
+
+const logger = Logger.get("WaterContentSensor");
+
+class WaterContentSensor {
+    async start() {
+        logger.info(`node-matter@${packageJson.version}`);
+
+        const deviceName = "Matter Sensor device";
+        const deviceType = DEVICE.HUMIDITY_SENSOR.code ;
+        const vendorName = "node-matter";
+        const passcode = 20202021;
+        const discriminator = 3840;
+        // product name / id and vendor id should match what is in the device certificate
+        const vendorId = new VendorId(0xFFF1);
+        const productName = "Matter Test DAC 0007";
+        const productId = 0X8000;
+
+        // Barebone implementation of the WaterContentMeasurement cluster
+        const waterContentMeasurementClusterServer = new ClusterServer(
+            WaterContentMeasurementCluster,
+            { measuredVAlue: false },
+            { measuredValue: 1, minMeasuredValue: 1, maxMeasuredValue: 1, tolerance: 1 },
```

Comment:
measuredValue: getIntParameter( "watercontent" )  ?? 0

This would avoid the next few lines of code...